opts/deno.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@synadiaorbit/opts",
+  "version": "1.0.0-1",
+  "exports": {
+    ".": "./mod.ts"
+  },
+  "publish": {
+    "exclude": ["./examples"]
+  },
+  "tasks": {
+    "clean": "rm -Rf ./coverage",
+    "test": "deno task clean & deno test --allow-all --parallel --reload --quiet --coverage=coverage",
```

Comment:
Is this supposed to be `&&` for "do left, then do right"?
Because at the moment, if this is interpreted in the regular shell fashion then it's doing the `deno task clean` in the background while it does the build, which seems likely to lead to difficult-to-reproduce states.

Reply:
`&&` runs the next command if the one to the left succeeds - see https://docs.deno.com/runtime/reference/cli/task/#boolean-lists

Reply:
Yes, but if you look at line 12, it was a single `&` not a `&&`, which is why I asked if it was supposed to be `&&`.  But I see you've since switched to instead make the `clean` task a dependency of this task, which solves the issue another way, so I'll mark this resolved.